apps/meteor/client/views/room/contextualBar/Info/hooks/useRoomActions.ts

```diff
@@ -8,75 +8,111 @@ import { useRoomHide } from './actions/useRoomHide';
 import { useRoomLeave } from './actions/useRoomLeave';
 import { useRoomMoveToTeam } from './actions/useRoomMoveToTeam';
 
-type RoomActions = {
+type UseRoomActionsOptions = {
```

Comment:
```suggestion
type RoomActionsProps = {
```

Reply:
Have you seen that is not embracing all the props? only the second one that fits better as options for the hook...

Reply:
`RoomActionsOptions` then I don't think Use fits for type naming since use is a convention to the hook name, its kinda weird

Reply:
I think this is more about a "naming conventions". I think it's easier to see that the type itself belongs to a hook, if it was a type exported everyone should presume the origin by the name.